proposals/lip-0063.md

```diff
@@ -218,50 +218,72 @@ ffffff
    <td>The address used to store all tokens of legacy accounts.
    </td>
   </tr>
+  <tr>
+   <td><code>Q96_ZERO</code>
+   </td>
+   <td>bytes
+   </td>
+   <td><code>''</code>
+   </td>
+   <td>The empty byte string, which represents the zero in Q96 representation.
+   </td>
+  </tr>
 </table>
 
 ### Mainnet Configuration
 
 #### Modules
 
-Lisk Core v4 will use the following modules:
+Lisk Core v4 will use the following modules, where the given order defines the module registration order:
 
-- [DPoS](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md)
-- [Auth](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0041.md)
-- [BFT](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0058.md)
-- [Validators](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0044.md)
-- [Random](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0046.md)
-- [Fee](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0048.md)
-- [Interoperability](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0045.md)
-- [Token](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0051.md)
-- [Legacy](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0050.md)
+1. [Auth](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0041.md)
+2. [Validators](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0044.md)
+3. [Token](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0051.md)
+4. [Fee](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0048.md)
+5. [Interoperability](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0045.md)
```

Comment:
I remember that Validators and Interoperability have a special status (system modules?) because they cannot be replaced and also that their hooks are always called first. Not sure if this is the case. Would this mean that these modules have to be registered first @shuse2 ?

Reply:
after moving to application/engine, notion `system modules` were removed.
We can register modules in particular order, but we currently don't have any assumption from the implementation side

Reply:
@shuse2 could you answer this one?

Reply:
The comment can be resolved from my side.